docs/intro/tutorial01.txt

```diff
@@ -52,9 +52,9 @@ code, then run the following command:
 
 .. console::
 
-   $ django-admin startproject mysite
+   $ django-admin startproject mysite djangotutorial
 
-This will create a ``mysite`` directory in your current directory. If it didn't
+This will create a ``djangotutorial`` directory in your current directory. If it didn't
```

Comment:
Per my testing, this is no longer true. When running the command above without having an existing folder named `djangotutorial`, I get this error:

```
$ django-admin startproject mysite djangotutorial
CommandError: Destination directory '/home/nessita/djangotutorial' does not exist, please create it first.
```

I think this intro needs to be adjusted to guide the user creating a `djangotutorial` folder as prerequisite so they don't get frustrated.

Reply:
Thank you for your observation. I have changed the text.